### PR TITLE
feature(fivetran_sdk): Add support for DataType Param

### PR DESCRIPTION
**Pull request by fivetran-abdulsalam**
Add DataTypeParam support and modified input file to support schemaDiffs

### CODE REVIEW

File: tools/destination-tester/README.md [hunk 1]
```diff
@@ -45,98 +45,113 @@ Here is an example input file named `input_1.json`:
...
+          "order_id": "INT",
+          "value": {
+            "type": "DECIMAL",
+            "precision": 6,
+            "scale": 3
+          }
         },
-        "campaign": {
-            "columns": {
-                "name": "STRING",
```
Comment (ediril) [inline tools/destination-tester/README.md:59]: Let's keep this one like this to show that byte length is optional

**Review by ediril — APPROVED** [review]
I requested one change but looks good otherwise

File: tools/destination-tester/README.md (hunk 1 above)
Comment (fivetran-abdulsalam) [reply tools/destination-tester/README.md:59]: done

**Review by manjutapali — APPROVED** [review]
Looks good.

**Pull request merged**